src/main/java/games/strategy/triplea/image/UnitImageFactory.java

```diff
@@ -253,6 +253,7 @@ private static String getBaseImageName(final UnitType type, final PlayerID id, f
           name.append("_ss");
         }
         if (TechTracker.hasRocket(id)) {
+          // do nothing
```

Comment:
If you look at the entire context of this method, this block can definitely be removed without effect.  However, I assumed the original author was trying to tell a future maintainer something...?